### PR TITLE
bf: revert deletion of upgrade info

### PR DESCRIPTION
**Pull request by ssalaues**
**What does this PR do, and why do we need it?**
Cherry-picks the doc that was deleted/reset in PR #609 which removed the upgrade info from 1.0 to 1.1

**Which issue does this PR fix?**
ZENKO-1708

**Special notes for your reviewers**:

### CODE REVIEW

File: docs/docsource/Installation/upgrade/upgrading_zenko.rst [hunk 1]
```diff
@@ -11,11 +11,16 @@ To upgrade Zenko:
 
 #. Unpack the .zip or .tar.gz file and navigate to Zenko/kubernetes/. 
 
-#. Enter this Helm commamnd, inserting your Zenko server's name
+#. Enter this Helm commamnd, inserting your Zenko server's release name
    :: 
 
       $ helm upgrade {{zenko-server-name}} ./zenko
 
+   If you are used a custom values be sure to reuse this file on upgrades.
```
Comment (JianqinWang) [inline docs/docsource/Installation/upgrade/upgrading_zenko.rst]: "used a custom values" -> "using custom values"

File: docs/docsource/Installation/upgrade/upgrading_zenko.rst [hunk 2]
```diff
@@ -29,6 +34,59 @@ To upgrade Zenko:
...
+Zenko 1.0.x versions use MongoDB version 3.4 which is upgraded to 3.6 in Zenko
+1.1.x versions. Upgrades using the previously mentioned commands will work
+however some newer features may not function by default.
+
+#. Run the typical upgrade command, inserting your Zenko server's release name
+   ::
+
+     $ helm upgrade {{zenko-release-name}} ./zenko
+
+   If you are used a custom values be sure to reuse this file on upgrades.
```
Comment (JianqinWang) [inline docs/docsource/Installation/upgrade/upgrading_zenko.rst]: same here

**Review by JianqinWang — APPROVED** [review]

File: docs/docsource/Installation/upgrade/upgrading_zenko.rst (hunk 1 above)
Comment (ssalaues) [reply docs/docsource/Installation/upgrade/upgrading_zenko.rst]: Both fixed. Thank you!

**Review by jonathan-gramain — APPROVED** [review]

**Comment by ssalaues** [comment]
@bert-e approve

**Pull request merged**